src/main/java/org/openstreetmap/atlas/geography/MultiPolyLine.java

```diff
@@ -88,12 +88,43 @@ public Rectangle bounds()
         return Rectangle.forLocations(locations);
     }
 
+    @Override
+    public boolean equals(final Object other)
+    {
+        if (!(other instanceof MultiPolyLine))
+        {
+            return false;
+        }
+        final MultiPolyLine otherItem = (MultiPolyLine) other;
+        final Iterator<PolyLine> iterator = this.iterator();
+        final Iterator<PolyLine> iteratorOther = otherItem.iterator();
+        while (iterator.hasNext() && iteratorOther.hasNext())
+        {
+            if (!iterator.next().equals(iteratorOther.next()))
```

Comment:
It looks like we are expecting the order of polylines to be the same in both objects. Do we want the equals to be this strict?

Reply:
ok, I can relax that.